docs/src/routes/docs/components/dropdown.svx

```diff
@@ -36,11 +36,21 @@ Toggleable, contextual overlays for displaying additional details or actions. Mu
     </DropdownShell>
     <DropdownShell let:toggle>
       <Button on:click={toggle}>
-        look up there
+        Look up there
```

Comment:
That was intentional, currently the styleguide of Attractions is to write buttons in lowercase. That is strange, however, and will change after the GrEaT reDeSIgN.

Reply:
HAHAHA OK, I didn't notice that. I Will change it back